resources/views/external-link.blade.php

```diff
@@ -1,7 +1,7 @@
 @props([
     'url',
     'text',
-    'class'     => 'link font-semibold inline break-words',
+    'class'     => 'link font-semibold inline',
```

Comment:
see https://github.com/ArkEcosystem/marketsquare.io/pull/2250 for similar comment